backend/src/game/dice.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Some of this information may not be bad for a docstring; however, I believe the dice should be an established class not just a module.

Reply:
When you do that though, then you have to use it as follows:
```d = Dice()
d.roll()
```

It's also not helpful as a class because we're not having to save state or anything. I had originally wrote it as a class and it just didn't feel right when I went to use it.